package.json

```diff
@@ -6,14 +6,21 @@
   "scripts": {
```

Comment:
Can the scripts be kept alphabetised please?

And there is no need for the `run` keyword to be used.